src/features/licenses/components/LicenseForm/index.jsx

```diff
@@ -144,6 +168,7 @@ export const LicenseForm = ({
           maxLength="255"
           value={fields.licenseName}
           onChange={handleInputChange}
+          disabled={disableLicenseNameIn}
```

Comment:
This is not a robust validation, is easy to fool, so would be a nice idea to add this validation in the information before send it this info.

Reply:
What do you mean with 'add this validation in the information'?

Reply:
What I mean by this is that as a user I can go to the devtools and disable that property, and then change the name. If the goal is to prevent the user from changing the catalog name under certain conditions, the function that sends the request with the information from the form should also validate if it meets the condition of sending the license name. If it doesn't, it shouldn't send it. I know, it's not the biggest security risk in the world, but it makes the system a little more robust. What do you think?

Reply:
In this case, if the user modifies the license name input, it's not a security risk, that is why user can edit it after. This disable is more for convenience, but it's likely they'll remove it later